### DISCUSSION TITLE
dispach to js

### DISCUSSION THREAD

**Discussion by sbc640964** [filamentphp/filament#7814] [Ideas]
Further to my question https://github.com/filamentphp/filament/discussions/7802

How can I create the following flow?

1. When a form is submitted a js action will be executed
2. The js operation will end successfully and close the form modal
3. Send a notification about success or failure

**Comment by pboivin** [comment]
What does it mean to close the form?

**Reply by sbc640964** [reply]
@pboivin Sorry, I meant modal

**Reply by pboivin** [reply]
I'm not sure but if you need to control the `submit` action and `close` action from JS, it could be simpler to create a custom modal instead.
https://filamentphp.com/docs/3.x/support/blade-components/modal
